RecoTracker/MkFitCore/standalone/xeon_scripts/init-gcc10-env.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-source /cvmfs/cms.cern.ch/slc7_amd64_gcc10/lcg/root/6.20.06-cms/etc/profile.d/init.sh
-export TBB_GCC=/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/tbb/2020_U2
+source /cvmfs/cms.cern.ch/slc7_amd64_gcc10/lcg/root/6.24.07-f52350f4e0b802edeb9a2551a7d00b92/etc/profile.d/init.sh
+export TBB_GCC=/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/tbb/v2021.4.0-d0152ca29055e3a1bbf629673f6e97c4
 # workaround for https://github.com/cms-sw/cmsdist/issues/5574
 # remove when we switch to a ROOT build where that issues is fixed
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$LIBJPEG_TURBO_ROOT/lib64
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$LIBJPEG_TURBO_ROOT/lib64
```

Comment:
should this part be dropped completely?
IIUC the issue was already fixed in 12_3_0_pre6 for which the readme is intended

Reply:
Fine with me, @slava77, will delete the comments.

Question: when I update the PR, should I also remove init-env.sh? It relies on gcc from the /cvmfs/cms.cern.ch/slc7_amd64_gcc820 tree. But I find that mkFit no longer compiles with the gcc 8.3.1 environment created by init-env.sh. Two issues:

1. `#include <vector>` is apparently missing from Hit.h - this can (should?) be fixed
2. TBB's include files cannot be found in the oneapi/tbb location, based on the TBB_GCC path - this also is (sort of) fixable

Details: if I use the same instructions in the new README, but I source init-env.sh rather than init-gcc10-env.sh, I get errors like this:
```
/home/users/slantz/mkFitFromCMSSW/RecoTracker/MkFitCore/interface/Hit.h:118:16: error: 'vector' in namespace 'std' does not name a template type
   typedef std::vector<MCHitInfo> MCHitInfoVec;
                ^~~~~~
/home/users/slantz/mkFitFromCMSSW/RecoTracker/MkFitCore/interface/Hit.h:118:11: note: 'std::vector' is defined in header '<vector>'; did you forget to '#include <vector>'?
```
The errors go away if I simply insert `#include <vector>` in Hit.h. Perhaps it inadvertently got left out somewhere in the grand rearrangement for CMSSW? I don't know why gcc-10 doesn't seem to care about this--maybe `#include <vector>` sneaks in some other way.

There are additional failures at compile time because various include statements now assume oneapi/tbb, but the slc7_amd64_gcc820 tree only provides versions of TBB prior to oneAPI. This can be fixed (or at least hacked around) by setting TBB_GCC to point to an installation of oneAPI TBB somewhere else.

Reply:
- I agree that that the include should be added.
- regarding gcc9 vs 10 .sh files; perhaps it makes sense to simply update init-env.sh and drop init-gcc10-env.sh? With the idea that the default has no name extension, while the new/experimental, or old will be named.

Reply:
Agree with @slava77 on this approach, pushing a new commit.